### DISCUSSION TITLE
Power Management: option to auto-suspend on low battery

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#334] [Ideas]
Currently, the power management has the option of shutting down and doing nothing if the battery is low. Please add a "suspend" option, because a laptop can stay suspended for several hours on the last few percents of battery.

![image](https://github.com/linuxmint/discussions/assets/57274935/f1b65597-8cfd-4c0a-8923-5b3e4765c81d)

**Comment by rcalixte** [comment]
There's already [an applet](https://cinnamon-spices.linuxmint.com/applets/view/255) that provides this functionality. It works in tandem with the existing power settings.
